udata_front/theme/gouvfr/templates/macros/sort_search.html

```diff
@@ -0,0 +1,30 @@
+{% macro sort_search(result, sorts) %}
+{% if sorts %}
+    {% if in_url('sort') %}
+        {% set current_sort = request.args.sort %}
+    {% endif %}
+{% endif %}
+<div class="fr-grid-row align-items-center">
+    <label for="sort-search" class="fr-col-auto fr-text--sm fr-m-0 fr-mr-1w">
+        {{_('Sort by:')}}
+    </label>
+    <div class="fr-col">
+        <select
+            id="sort-search"
+            class="fr-select"
+            data-select-sort
+            name="sort"
+        >
+            <option value="" {{selected(current_sort, Nil)}} disabled>
```

Comment:
The relevance option is disabled even if it isn't selected. We can remove the `disabled` attribute or make it conditional?